### PR TITLE
Add a CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by hfwang**
Describes the process of opening an issue, signing the CLA and our target response rate.

Borrowed liberally from our exisitng bug update SLO and other Google
contributing guides:
https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/CONTRIBUTING.rst
https://github.com/GoogleCloudPlatform/google-cloud-common/blob/master/contributing/readme.md
https://github.com/GoogleCloudPlatform/google-cloud-go/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by hfwang** [comment]
You can check out what it looks like in my fork: https://github.com/hfwang/cloudsql-proxy/blob/e2023488962642a2de00a15be536c0b058fc673d/CONTRIBUTING.md

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Contributing
+
+1. **Please sign one of the contributor license agreements below!**
+1. Fork the repo, develop and test your code changes, add docs.
+1. Make sure that your commit messages clearly describe the changes.
+1. Send a pull request.
+
```
Comment (kurtisvg) [inline CONTRIBUTING.md]: I think a TOC here would be good. `.. contents:: Table of Contents`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+## Opening an issue
+
+If you've tried everything in our [Cloud SQL Proxy Client
```
Comment (kurtisvg) [inline CONTRIBUTING.md]: I think we should reword this - we want it to be clear that this forum is for discussing/troubleshooting the proxy itself, and that any issues with Cloud SQL should open a support ticket.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+1. Go to the project issues page on GitHub.
+1. Click the green `New Issue` button located in the upper right corner.
+1. In the title field, write a single phrase that identifies your issue.
+1. In the main editor, describe your issue.
+1. Click the submit button.
+
+Thank you. We will attempt to categorize your issues with an estimate of the
+priority and issue type. We will do our best to comment on your issue within one
+business day. If we believe the issue is a genuine bug, we will try to respond
```
Comment (kurtisvg) [inline CONTRIBUTING.md]: Let's remove `If we believe the issue is a genuine bug,` -- It doesn't really address feature requests or documentation questions. We should use the closing of an issue to signal we aren't devoting any time to it.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+with regular updates based on its priority:
+
+* **Critical** respond and update daily, resolve with a week
```
Comment (kurtisvg) [inline CONTRIBUTING.md]: We should discuss guidelines on what issues fall under what categories.

**Pull request merged**